TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -563,9 +563,10 @@ public static void UnqueueCommand(string user, bool isWhisper, [Group(1)] bool u
 	public static void CallQueuedCommand(string user, bool isWhisper, [Group(1)] bool now, [Group(2)] string name)
 	{
 		name = name?.Trim();
+		name = name == null ? "" : name;
```

Comment:
I added this just to make sure that adding `name` to `TwitchGame.Instance.CallingPlayers.Values` wouldn't cause an exception. It also helps out on line 593. Feel free to remove it if it is not necessary.